beetsplug/smartplaylist.py

```diff
@@ -39,8 +44,12 @@ def __init__(self):
             'auto': True,
             'playlists': [],
             'forward_slash': False,
+            'prefix': u'',
+            'urlencode': False,
         })
 
+        # may contains user and password
+        self.config['prefix'].redact = True
```

Comment:
```suggestion
        self.config['prefix'].redact = True  # May contain username/password.

```

Preferring a blank line afterward because this is unrelated to the next two lines.